src/Ombi.Notifications/Agents/EmailNotification.cs

```diff
@@ -71,14 +71,24 @@ protected override bool ValidateConfiguration(EmailNotificationSettings settings
             return message;
         }
 
+        private async Task<string> LoadMessage(NotificationType type, NotificationOptions model, EmailNotificationSettings settings)
```

Comment:
Can we change the method name to something like `LoadPlainTextMessage`? 

It could be a bit confusing since there is another `LoadMessage` method on the `base` with different parameters that `LoadTemplate` ends up calling